specs/lfp-appeals-api-spec.json

```diff
@@ -0,0 +1,339 @@
+            "description": "ID Reference of the penalty against the company",
+            "in": "query",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Appeals resource retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/Appeal"
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Appeals not found"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Appeal"
+        ],
+        "summary": "Create a new appeal",
+        "operationId": "submitAppeal",
+        "parameters": [
+          {
+            "name": "ERIC-identity",
+            "in": "header",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "company_number",
+            "description": "number of the company filing the appeal",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Appeal"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "201": {
+            "description": "Appeal resource created",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string",
+                  "example": "60d8aa8d8fd5823a2d0108c5"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid appeal"
+          },
+          "401": {
+            "description": "User is not authorised to submit this appeal"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/companies/{company_number}/appeals/{id}": {
+      "get": {
+        "tags": [
+          "Appeal"
+        ],
+        "summary": "Get an appeal by ID",
+        "operationId": "getAppealById",
+        "parameters": [
+          {
+            "name": "company_number",
+            "description": "number of the company filing the appeal",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "id",
+            "description": "ID of the appeal in mongoDB",
```

Comment:
This shouldn't reveal implementation detail. Can it be changed to "ID of the appeal" please?